docs/Concepts.md

```diff
@@ -49,20 +49,20 @@ metadata:
 spec:
   inputs:
     resources:
-    - name: workspace
-      type: git
-      targetPath: go/src/github.com/knative/build-pipeline
+      - name: workspace
+        type: git
+        targetPath: go/src/github.com/knative/build-pipeline
   steps:
-  - name: unit-tests
-    image: golang
-    command: ['go']
-    args:
-    - 'test'
-    - './...'
-    workingDir: "/workspace/go/src/github.com/knative/build-pipeline"
-    env:
-    - name: GOPATH
-      value: /workspace/go
+    - name: unit-tests
+      image: golang
+      command: ["go"]
+      args:
+        - "test"
+        - "./..."
+      workingDir: "/workspace/go/src/github.com/knative/build-pipeline"
+      env:
+        - name: GOPATH
+          value: /workspace/go
```

Comment:
i am confused! i thought we you were horrified by this amount of indentation when you saw it in the WG @mattmoor - is this the right amount?

Reply:
I prefer less for sure (esp. with deep objects), but auto-fixable (and therefore enforceably consistent) feels like an even stronger bias.  I haven't explored this thing's knobs too much, but I'd be surprised if there were one for this (seems like it would be super niche to have a yaml-in-markdown indentation flag!).